src/material/schematics/ng-generate/tree/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.ts.template

```diff
@@ -84,7 +83,10 @@ export class <%= classify(name) %>Component {
   }
 
   /** Get the children for the node. */
-  getChildren(node: FileNode) {
-    return observableOf(node.children);
+  getChildren(node: FileNode): FileNode[] | null | undefined {
+    if (node.children === null || node.children === undefined) {
+      return node.children;
+    }
+    return node.children;
```

Comment:
It seems like both of the `if` clause outcomes will return the same value. You should be able to reduce it to just `return node.children`.